app/javascript/gobierto_data/webapp/components/commons/SavingDialog.vue

```diff
@@ -306,17 +349,43 @@ export default {
       const inputValueSplit = [...label]
       const inputValueLength = inputValueSplit.length
 
+      let newWidth = inputValueLength * 7.5
+      let maxWidth = 289.5
+      let minWidth = 200
+      newWidth = newWidth > 290 ? maxWidth : newWidth
+
+      //Check if the user is typing in the input of the queries or visualizations
+      let editorInputViz
+      if (this.$refs.inputText) {
+        editorInputViz = this.$refs.inputText.placeholder.includes(this.labelVisName) ? true : false
+      }
+
+      //In the Visualizations view, we've more space, so we take advantage of it by increasing the input name width.
+      if (this.$route.name === 'Visualization') {
+        maxWidth = maxWidth * 1.4
+        minWidth = minWidth * 1.4
+      } else if (editorInputViz) {
```

Comment:
This condition is so twisted. First off, `editorInputViz` should be initialized, in this case to `false`, but actually the real check is just `this.$refs.inputText.placeholder.includes(this.labelVisName)` which could be simplified like:
```
const { placeholder = [] } = this$ref.inputText || {};
....
else if (placeholder.includes(this.labelVisName)) {
....
}
```